doc/sphinx/source/build/project_build.rst

```diff
@@ -143,7 +143,7 @@ and generate executable file (.bin/.hex)
                .. image:: /source/build/stm32_exclude_files_option.png
                   :width: 600
 
-            * In order to choose STM32 platform in the firmware, select the "ACTIVE_PLATFORM" as "STM32_PLATFORM" in the app_config.h from the respective project.
+            * In order to choose STM32 platform in the firmware, select the "ACTIVE_PLATFORM" as "STM32_PLATFORM" in the app_config.h from the respective project. Alternately , add compiler flag "ACTIVE_PLATFORM == 2" for selecting stm32 platform.
```

Comment:
Should this be mentioned as "set the ACTIVE_PALTFORM" to the value defined for "STM32_PLATFORM" ?
There are some projects in which we've added the Value of STM32_PLATFORM as 1. It was suggested that way since STM32 Platform is where we get the max ODR.